src/address.rs

```diff
@@ -116,17 +103,18 @@ impl<A, Rc: RefCounter> Address<A, Rc> {
     /// })
     /// ```
     pub fn is_connected(&self) -> bool {
-        !self.sink.is_disconnected() && self.ref_counter.is_connected()
+        self.0.is_connected()
     }
 
     /// Returns the number of messages in the actor's mailbox.
     pub fn len(&self) -> usize {
-        self.sink.len()
+        todo!("Len")
```

Comment:
Still missing an implementation! :)

Reply:
This ties into the unresolved question on bounding - it can't get an impl till that's resolved